source/reference/beta-program/many-to-one.txt

```diff
@@ -0,0 +1,15 @@
+.. _c2c-beta-many-to-one:
+
+===========
+Many To One
```

Comment:
[question]

Should this be hyphenated like "Many-To-One"?

Reply:
Opting to update the title to be more specific.